implementations.md

```diff
@@ -15,25 +15,35 @@ This page contains an alphabetical list of projects and vendors that currently i
 [Ruby](https://github.com/elastic/apm-agent-ruby/blob/b68f1f12ae48a5c6e757241c65de97a98488ee6a/lib/elastic_apm/trace_context.rb),
 [JavaScript - Real User Monitoring](https://github.com/elastic/apm-agent-rum-js)
 
+
 ## Dynatrace
 **Website:** [dynatrace.com](https://www.dynatrace.com)
 
 **Implementations:**
 [All technologies supported by Dynatrace OneAgent](https://www.dynatrace.com/news/blog/distributed-tracing-with-w3c-trace-context-for-improved-end-to-end-visibility-eap/).
 
+
 ## Instana
 **Website:** [instana.com](https://www.instana.com)
 
 **Implementations:**
 [Node.js](https://github.com/instana/nodejs-sensor)
 Java
 
+## Jaeger
```

Comment:
```suggestion

## Jaeger
```

since you added an extra line everywhere else 😉

Reply:
thx - rebase conflict